src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -477,6 +477,13 @@ def _collect_module_maps(target):
       maps += module_maps
   return maps
 
+def _get_module_name(label):
```

Comment:
This should not be computed by the aspect, but rather propagated through providers. ObjcProvider should have the module_name key.